test/integration/targets/filters/tasks/main.yml

```diff
@@ -328,3 +328,27 @@
     - assert:
         that:
           - "combined.key2 == 'is_defined'"
+
+- debug: var=ansible_os_family
+- debug: var=ansible_distribution
+- debug: var=ansible_distribution_major_version
+- debug: var=ansible_distribution_version
+
+- set_fact:
+    sample_bcrypt_output: '$2b$12$123456789012345678912unE0GeDfx0eAsIsscM2jcQtjpIY2LVuy'
+    cafe_bcrypt_output: '$2b$12$123456789012345678912uDZsaKh/j7qZkKps5QEOGP4UfpxNa56S'
+    kryivka_bcrypt_output: '$2b$12$123456789012345678912utpRGnLLKUmQLWXHzHrg8U2Lwths9TNu'
+  when: ansible_distribution in ('OpenBSD', 'RedHat', 'FreeBSD', 'CentOS', 'Fedora', 'openSUSE Leap', 'Ubuntu')
+
+- set_fact:
+    sample_bcrypt_output: '$2a$12$123456789012345678912unE0GeDfx0eAsIsscM2jcQtjpIY2LVuy'
+    cafe_bcrypt_output: '$2a$12$123456789012345678912uDZsaKh/j7qZkKps5QEOGP4UfpxNa56S'
+    kryivka_bcrypt_output: '$2a$12$123456789012345678912utpRGnLLKUmQLWXHzHrg8U2Lwths9TNu'
+  when: ansible_distribution in ('MacOSX')
+
+- name: Test OpenBSD bcrypt algorithm (#54071)
+  assert:
+    that:
+      - "'{{ 'sample' | password_hash(hashtype='blowfish', salt='1234567890123456789123') }}' == '{{ sample_bcrypt_output }}'"
```

Comment:
> 07:03     "assertion": "'$2a$12$123456789012345678912unE0GeDfx0eAsIsscM2jcQtjpIY2LVuy' == '$2b$12$123456789012345678912unE0GeDfx0eAsIsscM2jcQtjpIY2LVuy'", 

Ubuntu 16.04 -> https://app.shippable.com/github/ansible/ansible/runs/136075/72/console

> 07:02 ERROR: Exception:
07:02 Traceback (most recent call last):
07:02   File "/root/ansible_testing/venv/lib/python2.7/site-packages/pip/_internal/cli/base_command.py", line 188, in main
07:02     status = self.run(options, args)
07:02   File "/root/ansible_testing/venv/lib/python2.7/site-packages/pip/_internal/commands/install.py", line 345, in run
07:02     resolver.resolve(requirement_set)
07:02   File "/root/ansible_testing/venv/lib/python2.7/site-packages/pip/_internal/legacy_resolve.py", line 196, in resolve
07:02     self._resolve_one(requirement_set, req)
07:02   File "/root/ansible_testing/venv/lib/python2.7/site-packages/pip/_internal/legacy_resolve.py", line 359, in _resolve_one
07:02     abstract_dist = self._get_abstract_dist_for(req_to_install)
07:02   File "/root/ansible_testing/venv/lib/python2.7/site-packages/pip/_internal/legacy_resolve.py", line 307, in _get_abstract_dist_for
07:02     self.require_hashes
07:02   File "/root/ansible_testing/venv/lib/python2.7/site-packages/pip/_internal/operations/prepare.py", line 215, in prepare_linked_requirement
07:02     finder, self.build_isolation,
07:02   File "/root/ansible_testing/venv/lib/python2.7/site-packages/pip/_internal/distributions/source.py", line 70, in prepare_distribution_metadata
07:02     reqs = self.req.pep517_backend.get_requires_for_build_wheel()
07:02   File "/root/ansible_testing/venv/lib/python2.7/site-packages/pip/_vendor/pep517/wrappers.py", line 71, in get_requires_for_build_wheel
07:02     'config_settings': config_settings
07:02   File "/root/ansible_testing/venv/lib/python2.7/site-packages/pip/_vendor/pep517/wrappers.py", line 162, in _call_hook
07:02     raise BackendUnavailable
07:02 BackendUnavailable

FreeBSD 11.1 -> https://app.shippable.com/github/ansible/ansible/runs/136075/64/console